front/app/components/ParticipationCTABars/ParticipationCTAContent/index.tsx

```diff
@@ -57,6 +57,8 @@ export const ParticipationCTAContent = ({
 }: Props) => {
   const theme = useTheme();
   const isSmallerThanPhone = useBreakpoint('phone');
+  const isSurveyPostingEnabled = currentPhase?.attributes.posting_enabled;
```

Comment:
This works for now, I think it would be nicer to handle this with action descriptors for consistency. But that will involve BE work because right now it seems like action descriptors don't work at all for native surveys